### DISCUSSION TITLE
混合

### DISCUSSION THREAD

**Discussion by xiatianPE** [hexo-theme-Acrylic/Hexo-Theme-Acrylic#57] [Q&A]
能否混合Butterfly原版使用，就是覆盖Butterfly原文件

**Comment by Marcus-zh** [comment]
不能覆盖的呢,因为我们在btf上面做了很多改动(~删屎山~),配置文件与btf有很多不同的地方,您可以去对比一下,文档暂未完善,还请多多指教

**Reply by xiatianPE** [reply]
那能否与Acrylic-Next共存

**Reply by Marcus-zh** [reply]
可以的,但是两个的配置文件不同,就是说不能共用一份配置文件,因为我们是两个不同的开发者分别负责的两个主题,难免会有差异

**Reply by xiatianPE** [reply]
难道会起冲突？

**Reply by Marcus-zh** [reply]
额,配置文件不用同一个就行
